app/controllers/users/registrations_controller.rb

```diff
@@ -14,12 +14,13 @@ def create
     if current_user
       alpaca = CreateRandomAlpaca.new.call
       alpaca.save
+      InitializeNewAlpacaSkills.new(Alpaca.find(alpaca.id)).call
       ownership = Ownership.new
       ownership.alpaca_id = alpaca.id
       ownership.user_id = current_user.id
       ownership.save
-      current_user.hay = 200
-      current_user.save
+      current_user.update_attributes(hay: 200)
```

Comment:
`update_attributes` skips validation. We should always use `update`, unless there is a specific problem, which requires skipping validation

Reply:
update_attribute skpis validation. update_attribute doesn't ;)
I tried with update, but it didn't worked

Reply:
My bad :) good job!